cypress/integration/TextInputLabs.spec.ts

```diff
@@ -0,0 +1,79 @@
+import * as h from '../helpers';
+
+const getTextInput = () => {
+  return cy.get(`[type="text"]`);
+};
+
+describe('TextInput', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Basic', 'Alert', 'Error'].forEach(story => {
+    context(`given the '${story}' story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Labs/Text Input/React', story);
+      });
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      context('when clicked', () => {
+        beforeEach(() => {
+          getTextInput().click();
```

Comment:
I think these helpers are fine, but I've been preferring Cypress Testing Library helpers:
```ts
cy.findByRole('textbox', { name: 'Email' })
```

Sometimes it is fine to not specify the accessible name:
```ts
cy.findByRole('textbox')
```

The downside is repeating, but the upside is a direct correlation to Cypress command log. `getTextInput` will not show on the Cypress command log while `findByRole` with it's arguments will.

Reply:
Agree, I just straight copied these tests from the non-labs component. Didn't want to mess with them too much until we had a solid agreement of what the spec of this component should be.